### ISSUE TITLE
[BUG]: Monsters not spawning

### ISSUE THREAD

**Issue by VanilaBOI** [XuuXiao/GiantSpecimens#19]
### Installation Method

r2modman / r2mm

### What happened?

Even tough I turned up the spawn rates for the red wood giant on "exp" to 999, it won't spawn.

### What did you do?

I thought at first it was advanced company's way of handling enemy weights but after removing it I thought it was LLL but after removing LLL it still happend.
I feel like I downloaded something that just f'd my game up and I need to reinstall it all together.

v62

### Version

v50

### What OS are you running?

Windows 10

### List of mods used

Giant Specimens

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by VanilaBOI** [comment]
The weird thing is that I think it worked at some point because I was able to adjust the spawn weights of the giants through advanced company but now they don't show up on advanced company or the LLL config.

**Comment by XuuXiaolan** [comment]
can you send a screenshot of your config?

**Comment by VanilaBOI** [comment]
## Settings file was created by plugin TheGiantSpecimens v2.2.1
## Plugin GUID: TheGiantSpecimens

[Enemy Options]

## Config the enemy power of the Redwood Giant
# Setting type: Single
# Default value: 1
Redwood Giant | Enemy Power = 1

## Config the enemy power of the Driftwood Giant
# Setting type: Single
# Default value: 1
DriftWood Giant | Enemy Power = 1

## Enables/Disables the Zeus mode.
# Setting type: Boolean
# Default value: false
RedWood Giant | Zeus Mode = false

## Enables/Disables the eating of old birds.
# Setting type: Boolean
# Default value: true
RedWood Giant | Eating Palette = true

## Configure the range of the scream of the Driftwood Giant
# Setting type: Single
# Default value: 20
DriftWood Giant | Scream Range = 20

## Multiplier in Forest Keeper spawnrate after the RedWood Giant spawns.
# Setting type: Int32
# Default value: 2
ForestKeeper Multiplier = 2

## Multiplier in Driftwood spawnrate after the RedWood Giant spawns
# Setting type: Int32
# Default value: 2
Driftwood Multiplier = 2

## Enables/Disables the spawning of the RedWood Giant (sets rarity to 0 if false on all moons)
# Setting type: Boolean
# Default value: true
RedWood Giant | Enabled = true

## Spawn Weight of the RedWood Giant in all moons, Feel free to add to it any moon, just follow the format (also needs LLL installed for LE moons to work with this config).
# Setting type: String
# Default value: Modded:100,ExperimentationLevel:50,AssuranceLevel:100,VowLevel:200,OffenseLevel:100,MarchLevel:200,RendLevel:200,DineLevel:100,TitanLevel:200,Adamance:100,Embrion:150,Artifice:200,Auralis:150
RedWood Giant | Spawn Weight. = Modded:100,ExperimentationLevel:999,AssuranceLevel:100,VowLevel:200,OffenseLevel:100,MarchLevel:200,RendLevel:200,DineLevel:100,TitanLevel:200,Adamance:100,Embrion:150,Artifice:200,Auralis:150

## Default walking speed of the RedWood Giant, (Chase speed is 4*Walking Speed) I recommend 1.5 to 3.
# Setting type: Single
# Default value: 2
RedWood Giant Speed = 2

## Distance of the Forest Keeper to the ship that stops the RedWoodGiant from chasing them, I recommend 0 to 15f (values are completely untested).
# Setting type: Single
# Default value: 10
RedWood Giant Targetting Range | Ship = 10

## Distance from which the RedWood Giant is able to see the Forest Keeper, I recommend 30f or more.
# Setting type: Single
# Default value: 50
RedWood Giant Targetting Range | Forest Keeper = 50

## Decides what the default colour of the footsteps is using a hexcode, default is grey (Invalid hexcodes will default to Grey), keep blank to use Modded set colours set by me for different moons, don't forget to include the hashtag in config.
# Setting type: String
# Default value: #808080
RedWood Giant | Footstep Colour = #808080

## Enables/Disables the spawning of the driftwood (sets rarity to 0 if false on all moons)
# Setting type: Boolean
# Default value: true
Driftwood | Enabled = true

## Rarity of driftwood appearing on every moon
# Setting type: String
# Default value: Modded:100,ExperimentationLevel:75,AssuranceLevel:50,VowLevel:150,OffenseLevel:50,MarchLevel:175,RendLevel:125,DineLevel:125,TitanLevel:150,Adamance:100,Embrion:150,Artifice:200
Driftwood | Rarity = Modded:100,ExperimentationLevel:75,AssuranceLevel:50,VowLevel:150,OffenseLevel:50,MarchLevel:175,RendLevel:125,DineLevel:125,TitanLevel:150,Adamance:100,Embrion:150,Artifice:200

[Scrap Options]

## Enables/Disables the Heart (Currently looks like an LGU Sample) from spawning on death.
# Setting type: Boolean
# Default value: true
DriftWood Giant | Heart = true

## Enables/Disables the Heart from spawning on death.
# Setting type: Boolean
# Default value: true
RedWood Giant | Heart = true

## Enables/Disables the Plushie from appearing ingame.
# Setting type: Boolean
# Default value: true
RedWood Giant | Plushie = true

## Rarity of Whistle scrap appearing on every moon (set to Modded:0,Vanilla:0 if you'd like for it to not spawn).
# Setting type: String
# Default value: Modded:5,ExperimentationLevel:5,AssuranceLevel:5,VowLevel:5,OffenseLevel:5,MarchLevel:5,RendLevel:5,DineLevel:5,TitanLevel:5,Adamance:5,Embrion:5,Artifice:5
Whistle Scrap | Rarity = Modded:5,ExperimentationLevel:5,AssuranceLevel:5,VowLevel:5,OffenseLevel:5,MarchLevel:5,RendLevel:5,DineLevel:5,TitanLevel:5,Adamance:5,Embrion:5,Artifice:5

## Rarity of redwood plushie appearing on every moon (set to Modded:0,Vanilla:0 if you'd like for it to not spawn).
# Setting type: String
# Default value: Modded:5,ExperimentationLevel:5,AssuranceLevel:5,VowLevel:5,OffenseLevel:5,MarchLevel:5,RendLevel:5,DineLevel:5,TitanLevel:5,Adamance:5,Embrion:5,Artifice:5
RedWood Giant Scrap | Rarity = Modded:5,ExperimentationLevel:5,AssuranceLevel:5,VowLevel:5,OffenseLevel:5,MarchLevel:5,RendLevel:5,DineLevel:5,TitanLevel:5,Adamance:5,Embrion:5,Artifice:5

## Enables/Disables the whistle showing up
# Setting type: Boolean
# Default value: true
Whistle Scrap | Enabled = true

## Rarity of driftwood plushie appearing on every moon (set to Modded:0,Vanilla:0 if you'd like for it to not spawn).
# Setting type: String
# Default value: Modded:5,ExperimentationLevel:5,AssuranceLevel:5,VowLevel:5,OffenseLevel:5,MarchLevel:5,RendLevel:5,DineLevel:5,TitanLevel:5,Adamance:5,Embrion:5,Artifice:5
Driftwood Scrap | Rarity = Modded:5,ExperimentationLevel:5,AssuranceLevel:5,VowLevel:5,OffenseLevel:5,MarchLevel:5,RendLevel:5,DineLevel:5,TitanLevel:5,Adamance:5,Embrion:5,Artifice:5

[Shop Options]

## Enables/Disables the whistle showing up in shop
# Setting type: Boolean
# Default value: true
Whistle Item | Enabled = true

## Cost of Whistle
# Setting type: Int32
# Default value: 111
Whistle Item | Cost = 111

**Comment by VanilaBOI** [comment]
I really don't get anything
today testing it again, they both appeared in the monster list of advanced company so I adjusted the red wood giant spawn weight on exp to 999 to see if I can get him to spawn there but I only got dogs.
Now since I wasn't satisfied with that outcome I cranked the spawn weight of drift to 999 and red 100 on march to see if the 999 wasn't valid.
And wouldn't you know I got 2 drift spawns through Imperium telling me.
